doc/ranger.pod

```diff
@@ -229,14 +229,11 @@ To enable this feature, set the option C<preview_images_method> to urxvt-full.
 =head3 kitty
 
 This only works on Kitty. It requires PIL (or pillow) to work.
+It is able to automatically work over network,
```

Comment:
Nitpick: I'd stick closer to the original phrasing. "Allows remote image previews, for example in an ssh session." I'm not sure the performance implications should be in the manpage but I could be convinced otherwise.